packages/sdks-tests/src/tests/helpers/index.ts

```diff
@@ -178,3 +180,6 @@ export const checkIfIsHydrationErrorMessage = (_text: string) => {
 export const getClassSelector = (className: string) => {
   return isRNSDK ? `[data-class*=${className}]` : `.${className}`;
 };
+
+const __filename = fileURLToPath(import.meta.url);
+export const __dirname = path.dirname(__filename);
```

Comment:
can we name these something other than `__filename` and `__dirname`? Since it's a reserved global variable name in CJS node.js, it can confuse us down the road

Reply:
yes! should we name them `current_filename` and `current_dirname` ?

Reply:
sure, `currentFilename` and `currentDirname`